genepy/rna/__init__.py

```diff
@@ -422,6 +419,101 @@ def mergeSplicingVariants(df, defined='.'):
   df = df.drop(index=todrop)
   return df
 
+<<<<<<< HEAD
```

Comment:
it seemed you pushed the conflict..
did you edit slamdunk? if no you can just delete HEAD, otherwise you have to apply you edits (which are in the HEAD version to the upstream/master and then remove HEAD.

BELOW IS HEAD VERSION